sql/migrate/lex.go

```diff
@@ -174,10 +174,15 @@ func (l *lex) addPos(p int) {
 }
 
 func (l *lex) skipQuote(quote rune) error {
+	pos := l.pos
 	for {
 		switch r := l.next(); {
 		case r == eos:
-			return fmt.Errorf("unclosed quote %q", quote)
+			end := pos + 15
```

Comment:
Why this number? Can you please put some comment?

Reply:
Arbitrary number to provide some more feedback to where the unclosed quote is

What do you think is a reasonable length?

Reply:
From the tests it looks good . Just put it in a const with some descriptive name with some comment

Reply:
Position is not that good. Change it to line number + column number. Then, adding this string prefix is not needed.